crypto/keys/mintkey/mintkey.go

```diff
@@ -17,10 +17,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/keys/keyerror"
 )
 
-const (
-	blockTypePrivKey = "ETHERMINT PRIVATE KEY"
-	blockTypeKeyInfo = "ETHERMINT KEY INFO"
-	blockTypePubKey  = "ETHERMINT PUBLIC KEY"
+var appName = "ETHERMINT"
```

Comment:
you can actually just revert these changes, this file actually isn't needed anymore. Was needed before when there was an Ethermint keybase